script/bootstrap

```diff
@@ -60,6 +60,20 @@ setup_gitconfig () {
   fi
 }
 
+setup_git_symlinks () {
+  if ! [ $(readlink ~/.gitconfig) = */.dotfiles/developer/common/git/gitconfig.symlink ] &&
+   [ $(readlink ~/.gitconfig.local) = */.dotfiles/developer/common/git/gitconfig.local.symlink ] &&
+   [ $(readlink ~/.gitignore) = */.dotfiles/developer/common/git/gitignore.symlink ]
+  then
+    echo "Symlinks correct"
+  else
+    echo "Updating git symlinks.."
+    ln -s -f ~/.dotfiles/developer/common/git/gitconfig.symlink ~/.gitconfig
```

Comment:
You can combine the flags into one sequence: `ln -sf ...`

Reply:
BTW, can't you use the `link_files` function instead?

Reply:
check!